descriptors/ModuleDescriptor-template.json

```diff
@@ -629,7 +629,8 @@
             "proxiesfor.collection.get",
             "circulation.rules.notice-policy.get",
             "circulation-storage.patron-notice-policies.item.get",
-            "patron-notice.post"
+            "patron-notice.post",
+            "circulation-storage.cancellation-reasons.item.get"
```

Comment:
Please add the cancellation reasons interfaces to the set of dependencies for the module

Reply:
Done